.github/workflows/quick_deploy.yml

```diff
@@ -0,0 +1,21 @@
+
+name: Quick deploy trough code push
+
+on:
+  push:
+    branches: [master]
+
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Detox Tests
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.head_commit.message, 'skip code-push') }}
```

Comment:
condition to run **quick release** = `<NOT release>` && `<NOT PR merged>`

`<NOT PR merged>` = <commit message **NOT ends with** _`/\(\d+\)/`_>
`<NOT release>` = <commit message **NOT contains** _'release'_>

Reply:
you could keep github.event.pull_request.merged == true for 'PR merged' condition